Rewords digitizing snapping options fixes #1468 (#4289)
qgis/QGIS-Documentation: source/docs/user_manual/working_with_vector/editing_geometry_attributes.rst

================================================

For an optimal and accurate edit of the vector layer geometries, we need to set
an appropriate value of snapping tolerance and search radius for features
vertices.

.. index:: Snapping tolerance

Snapping tolerance
------------------

When you add a new vertex or move an existing one, snapping tolerance is the
distance QGIS uses to search for the closest vertex or segment you are
trying to connect to. If you aren't within the snapping tolerance, QGIS
will leave the vertex where you release the mouse button, instead of snapping
it to an existing vertex or segment.

The snapping tolerance setting affects all tools that work with tolerance.

You can enable or disabled snapping by using the |snapping| :sup:`Enable
snapping` button on the :guilabel:`Snapping Toolbar` or pressing :kbd:`S` at any

> time while you're on the map view. The toolbar is also very convenient to

Reviewer: ```suggestion
time while you're on the map view. This toolbar is also very convenient to
```

> quickly configure the snapping mode, tolerance value, and units. Nevertheless,

Reviewer: ```suggestion
quickly configure the snapping mode, tolerance value, and units.
```

> the snapping configurations can be set in :menuselection:`Project --> Snapping

Reviewer: ```suggestion
The snapping configuration can also be set in :menuselection:`Project --> Snapping
```

Options...`.

There are three options to select the layer(s) to snap to:

* :guilabel:`All layers`: quick setting for all visible layers
  in the project so that the pointer snaps to all vertices and/or segments.
  In most cases, it is sufficient to use this snapping mode, but beware when

>   using it on projects with too many vector layers, as it may cause

Reviewer: ```suggestion
  using it on projects with many vector layers, as it may cause
```

  slowness.
* :guilabel:`Current layer`: only the active layer is used, a convenient way

>   to ensure topology within the layer being edited

Reviewer: ```suggestion
  to ensure topology within the layer being edited.
```

* :guilabel:`Advanced Configuration`: allows you to enable and adjust snapping
  mode and tolerance on a layer basis (see figure_edit_snapping_). If you need
  to edit a layer and snap its vertices to another, ensure the target layer
  is checked and increase the snapping tolerance to a higher value.

>   Furthermore, snapping will not occur to a layer that is not checked in the

Reviewer: ```suggestion
  Snapping will not occur to a layer that is not checked in the
```

  snapping options dialog.

As for snapping mode, you can select between ``To vertex``, ``To segment``, and
``To vertex and segment``.

The tolerance values can be set either in the project's ``map units`` or in
``pixels``. The advantage of choosing ``pixels`` is that it keeps the snapping
constant at different map scales. 10 to 12 pixels is normally a good value, but

> it may depend on the DPI of your screen. On the other hand, using map units

Reviewer: ```suggestion
it depends on the DPI of your screen. Using map units
```

allows the tolerance to be related to real ground distances. For example, if you
have a minimum distance between elements, this option can be useful to ensure
that you don’t add vertexes too close to each other.

.. _figure_edit_snapping:

.. figure:: img/editProjectSnapping.png
   :align: center

   Snapping options (Advanced Configuration mode)

.. note::

   By default, only visible features (the features whose style is displayed,
   except for layers where the symbology is "No symbols") can be snapped. You
   can enable the snapping on invisible features by checking |unchecked|

>    :guilabel:`Enable snapping on invisible features` under

Reviewer: ```suggestion
   :guilabel:`Enable snapping on invisible features` under the
```

   :menuselection:`Settings --> Options --> Digitizing` tab.

.. tip:: **Enable snapping by default**

>    You can set snapping to be enabled by default on all new projects in

Reviewer: ```suggestion
   You can set snapping to be enabled by default on all new projects in the
```

   :menuselection:`Settings --> Options --> Digitizing` tab. You can also set
   the default snapping mode, tolerance value, and units, which will populate
   the :guilabel:`Snapping Options` dialog.

.. index:: Search radius

Search radius
--------------

Search radius is the distance QGIS uses to ``search`` for the closest vertex you
are trying to select when you click on the map. If you aren't within the search
radius, QGIS won't find and select any vertex for editing. The search radius for

> vertex edits can be defined under :menuselection:`Settings -->` |options|

Reviewer: ```suggestion
vertex edits can be defined under the :menuselection:`Settings -->` |options|
```
